.deepsource.toml

```diff
@@ -0,0 +1,10 @@
+version = 1
```

Comment:
this file, unless used in our CICD, should not be part of the PR

Reply:
DeepSource uses this `.deepsource.toml` file to continuously analyze every commit and pull request for code quality checks. I have linked the report of the DeepSource analysis above in my PR. There are plenty of other issues in relation to Bug Discovery, Performance and Anti-Patterns which you would be interested to take a look at.